x-pack/plugins/cloud_security_posture/public/pages/configurations/latest_findings/use_latest_findings.ts

```diff
@@ -74,7 +74,7 @@ export const getFindingsQuery = (
             },
           },
         ],
-        must_not: mutedRulesFilterQuery,
+        must_not: [...(query?.bool?.must_not ?? []), ...mutedRulesFilterQuery],
```

Comment:
@JordanSh this fixes the issue with the Non matching grouping such as "no cloud account" and "no kubernetes clusters"